cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java

```diff
@@ -378,7 +388,12 @@ private static String createOneSql(Map.Entry<String, ArrayList<Integer>> entry,
         continue;
       }
       if (timeseriesToType.get(headInfo.get(colIndex.get(j))).equals(STRING_DATA_TYPE)) {
-        sbd.append(", \'").append(data[colIndex.get(j) + 1]).append("\'");
+        if ((data[colIndex.get(j) + 1].startsWith("'") && data[colIndex.get(j) + 1].endsWith("'")) ||
```

Comment:
could you add some comment...  give some example

Reply:
ok